viseron/domains/face_recognition/__init__.py

```diff
@@ -70,6 +73,11 @@
             default=DEFAULT_SAVE_FACES,
             description=DESC_SAVE_FACES,
         ): bool,
+        vol.Optional(
```

Comment:
Since the `use_subjects` is specific to the `compreface` integration it should only be included there and not here in the base class

Reply:
3rd commit addresses this.